src/torch_ucc_init_oss.cpp

```diff
@@ -0,0 +1,4 @@
+#include <torch/python.h>
+
+PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
```

Comment:
I still need this so that I can get the filename of the shared object with
```
python -c "import inspect; import torch; import torch_ucc_oss; print(inspect.getfile(torch_ucc_oss))"
```

Reply:
Thanks much @zasdfgbnm - pl give me some time to test it - will let u know soon